agent/src/test/java/org/openjdk/jmc/agent/converters/test/GurkMultiDefaultConverter.java

```diff
@@ -62,4 +64,36 @@ public static int convert(Integer value) {
 	public static int convert(Long value) {
 		return value.intValue();
 	}
+
+	public static String convert(boolean b) {
+		return Boolean.toString(b);
+	}
+
+	public static String convert(byte b) {
+		return Byte.toString(b);
+	}
+
+	public static String convert(short s) {
+		return Short.toString(s);
+	}
+
+	public static String convert(char c) {
+		return "" + c;
+	}
+
+	public static long convert(int i) {
+		return i;
+	}
+
+	public static int convert(float f) {
+		return (int) f;
```

Comment:
Nit: not that it matters much in a test, but Math.round(f) might be a nicer default conversion.